File: src/1. Layers/1.3 Domain/Well.Domain/ValueObjects/Damage.cs
namespace PH.Well.Domain.ValueObjects
{
    using Enums;

    public class Damage
    {
        public int JobDetailId { get; set; }

        public int Quantity { get; set; }

        public int JobDetailSourceId { get; set; }

        public int JobDetailReasonId { get; set; }

        public int DamageActionId { get; set; }

        public DeliveryAction DamageAction => (DeliveryAction) DamageActionId;
    }
}

File: src/1. Layers/1.1 Presentation/Well.Dashboard/app/delivery/model/damage.ts
export class Damage
{

    constructor(
        index: number,
        quantity: number,
        jobDetailReasonId: number,
        jobDetailSourceId: number,
        damageActionId: number)
    {
        this.index = index;
        this.quantity = quantity;
        this.jobDetailReasonId = jobDetailReasonId;
        this.jobDetailSourceId = jobDetailSourceId;
        this.damageActionId = damageActionId;
    }

    public index: number;
    public quantity: number;
    public jobDetailReasonId: number;
    public jobDetailSourceId: number;
    public damageActionId: number;
    public jobDetailReason: string;
    public jobDetailSource: string;
    public damageAction: string;
}